File: 05_Recursion/Tree_Recursion/main.cpp
#include <stdio.h>

void TreeRecursion(int n) {
    if (n > 0) {
       printf("%d\n", n);
       TreeRecursion(n - 1);
       TreeRecursion(n - 1);
    }
}
int main() {
    int x = 3; 
    TreeRecursion(x);
}

File: 05_Recursion/Tree_Recursion/main.go
package main
import "fmt"

func TreeRecursion(n int) {
	if n > 0 {
		fmt.Printf("%d\n", n)
		TreeRecursion(n - 1)
		TreeRecursion(n - 1)
	}
}
func main() {
	x := 3
	TreeRecursion(x)
}


